hnn_core/cell_response.py

```diff
@@ -90,6 +95,10 @@ def __init__(self, spike_times=None, spike_gids=None, spike_types=None,
             spike_types = list()
         if times is None:
             times = list()
+        if vsec is None:
+            vsec = list()
+        if isec is None:
+            isec = list()
```

Comment:
Maybe I'm missing something but in line 129 following we set self._vsec = list(), are 98 and following therefore redundant?
Also, do we need a vsec and isec input as I don't think we use it?

Reply:
A good observation! I think my original motivation was to help with reloading back in data in the future, but you're right this should be removed.